ConfigSpace/hyperparameters.pyx

```diff
@@ -1407,11 +1407,12 @@ cdef class NormalIntegerHyperparameter(IntegerHyperparameter):
 
 cdef class CategoricalHyperparameter(Hyperparameter):
     cdef public tuple choices
-    cdef public tuple probabilities
+    cdef public list weights
```

Comment:
```suggestion
    cdef public tuple weights
```

Reply:
@mfeurer Had to make one more fix in order to make this work, but now it does!